pymeasure/instruments/rigol/rigol_dg800.py

```diff
@@ -0,0 +1,167 @@
+        map_values=True,
+    )
+
+    sine = Channel.setting(
+        ":SOUR{ch}:APPL:SIN %f,%f,%f,%f",
+        """
+        Set the waveform generator to output a sine of specified parameters
+        :param freq: (int) The frequency of the sine in Hz
+        :param ampl: (float) The peak to peak amplitude of the sine in V
+        :param offset: (float) The DC offset of the sine in V
+        :param phase: (float) The phase offset of the sine in degrees
+        """,
+    )
+    square = Channel.setting(
+        ":SOUR{ch}:APPL:SQU %f,%f,%f,%f",
+        """
+        Set the waveform generator to output a square wave of specified parameters
+        :param freq: (int) The frequency of the wave in Hz
+        :param ampl: (float) The peak to peak amplitude of the wave in V
+        :param offset: (float) The DC offset of the wave in V
+        :param phase: (float) The phase offset of the wave in degrees
+        """,
+    )
+    triangle = Channel.setting(
+        ":SOUR{ch}:APPL:RAMP %f,%f,%f,%f",
+        """
+        Set the waveform generator to output a triangle wave of specified parameters
+        :param freq: (int) The frequency of the wave in Hz
+        :param ampl: (float) The peak to peak amplitude of the wave in V
+        :param offset: (float) The DC offset of the wave in V
+        :param phase: (float) The phase offset of the wave in degrees
+        """,
+    )
+    pulse = Channel.setting(
+        ":SOUR{ch}:APPL:PULS %f,%f,%f,%f",
+        """
+        Set the waveform generator to output a pulse wave of specified parameters
+        Note that the duty cycle is specified using the duty_cycle method.
+        :param freq: (int) The frequency of the wave in Hz
+        :param ampl: (float) The peak to peak amplitude of the wave in V
+        :param offset: (float) The DC offset of the wave in V
+        :param phase: (float) The phase offset of the wave in degrees
+        """,
+    )
+
+    duty_cycle = Channel.control(
+        ":SOUR{ch}:PULS:DCYC?",
+        ":SOUR{ch}:PULS:DCYC %f",
+        """
+    Control the duty cycle of pulses out of the waveform generator.
+    :param duty_cycle: (float) The pulse's duty cycle
+    """,
+    )
+
+    pulse_width = Channel.control(
+        ":SOUR{ch}:PULS:WIDT?",
+        ":SOUR{ch}:PULS:WIDT %f",
+        """
+    Control the pulse width of pulses out of the waveform generator.
+    :param duty_cycle: (float) The pulse's width in seconds
+    """,
+    )
+
+    noise = Channel.setting(
+        ":SOUR{ch}:APPL:NOIS %f,%f",
+        """
+        Set the waveform generator to output noise of specified parameters
+        :param ampl: (float) The peak to peak amplitude of the noise in V
+        :param offset: (float) The DC offset of the noise in V
+        """,
+    )
+    dc = Channel.setting(
+        ":SOUR{ch}:APPL:DC 1,1,%f",
+        """
+    Set the waveform generator to output a DC voltage of specified parameters
+    :param dc: (float) The output DC voltage in V
```

Comment:
you have just a single parameter, so you do not need to state it as "param":
```suggestion
    Set the waveform generator to output a DC voltage  in V.
```